src/Symfony/Component/Serializer/Encoder/ChainEncoder.php

```diff
@@ -57,6 +57,8 @@ public function supportsEncoding($format)
      *
      * @param string $format
      *
+     * @throws RuntimeException if no encoder is found
```

Comment:
-1 for this. We don't care about what happens for unsupported formats anyway (the code is free to throw what it wants when it already avocated it does not support it)